tests/test_postgresql.py

```diff
@@ -71,13 +72,13 @@ def test_rand_postgres_port(postgresql_rand):
 
 @pytest.mark.parametrize('_', range(2))
 def test_postgres_terminate_connection(
-        postgresql, _):
+        postgres10, _):
```

Comment:
This seems to have been using postgres 9.x before. The `backend_type` column was introduced in version 10, so here we make sure that it'll always use at least 10